src/bundles/game/functions.ts

```diff
@@ -0,0 +1,1390 @@
+/**
+ * Game library that translates Phaser 3 API into Source.
+ *
+ * More in-depth explanation of the Phaser 3 API can be found at
+ * Phaser 3 documentation itself.
+ *
+ * For Phaser 3 API Documentation, check:
+ * https://photonstorm.github.io/phaser3-docs/
+ *
+ * @module game
+ * @author Anthony Halim
+ * @author Chi Xu
+ * @author Chong Sia Tiffany
+ * @author Gokul Rajiv
+ */
+
+/* eslint-disable @typescript-eslint/no-unused-vars, consistent-return, @typescript-eslint/no-shadow */
+import {
+  GameObject,
+  GameParams,
+  ObjectConfig,
+  RawContainer,
+  RawGameElement,
+  RawGameObject,
+  RawInputObject,
+} from './types';
+
+/** @hidden */
+export default function gameFuncs(_params: GameParams) {
+  const {
+    scene,
+    phaser,
+    preloadImageMap,
+    preloadSoundMap,
+    preloadSpritesheetMap,
+    remotePath,
+    screenSize,
+    createAward,
+  } = _params;
```

Comment:
Actually can put GameParams under a field in `__params`? Something like `__params.game`